src/emucore/Console.hxx

```diff
@@ -136,7 +136,6 @@ class Console
     */
     std::string getFormat() const { return myDisplayFormat; }
 
-  private:
```

Comment:
Not a mistake, this makes `getFrameRate()` public. This is required for `ScreenSDL`.